### DISCUSSION TITLE
How to compile the source code

### DISCUSSION THREAD

**Discussion by qq735-personal** [KadeArchive/Kade-Engine#2310] [Q&A]
I am making a mod in this engine and would like to compile the source code.
My computer sucks so I can't use the original game.
H E L P

**Comment by Riskerman21** [comment]
give us a screenshot? be more specific?

**Reply by qq735-personal** [reply]
What would I give a screenshot on
I mean to COMPILE the engine. So I can edit the code

**Reply by Riskerman21** [reply]
idk what step you are stuck on so ama give you this
https://www.youtube.com/watch?v=CKNfu8G9EOc
